fairlearn/metrics_plots/plot_disparities_in_performance.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from sklearn.metrics import accuracy_score
+from fairlearn.metrics import group_summary, false_positive_rate, false_negative_rate
+
+
+def plot_disparities_in_performance(y_true, y_pred, sensitive_features):
+    """Plot disparities in performance for a binary classifier across a single sensitive feature.
+
+    This helps visualize differences in overall prediction error, as well as false positive
+    and false negative error rates across the values of a single sensitive attribute
+    (eg, group membership).
+
+    Parameters
+    ----------
+    y_true : array-like
+        The list of true values
+
+    y_pred : array-like
+        The list of predicted values
+
+    sensitive_features : array-like
+        the sensitive attributes
+
+    Returns
+    -------
+    None
+        The function renders a matplotlib plot
+    """
+    # compute
+    accuracy_summary = group_summary(
+        accuracy_score,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    fp_summary = group_summary(
+        false_positive_rate,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    fn_summary = group_summary(
+        false_negative_rate,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    sensitive_values = sensitive_features.unique()
+    overpredictions = []
+    underpredictions = []
+    for sensitive_value in sensitive_values:
+        overpredictions.append(fp_summary['by_group'][sensitive_value])
+        underpredictions.append(fn_summary['by_group'][sensitive_value])
+    disparity = abs(
+        accuracy_summary['by_group'][sensitive_values[0]]
+        - accuracy_summary['by_group'][sensitive_values[1]])
+
+    # chart text for localization
+    title_text = 'Disparity in performance'
```

Comment:
These are not really localisable if they're embedded in the method. They need to be constants at the top of the file.

Reply:
Will there be localization? If so, I don't think it needs to be in the first version.